src/main/java/uk/gov/companieshouse/acsp/model/AMLSupervisoryBodies.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.companieshouse.acsp.model;
+
+public enum AMLSupervisoryBodies {
+    SOLE_TRADER("Sole Trader"),
```

Comment:
AMLSupervisory bodies would be something like HMRC, FDA etc. check with Benedict to get the list.